src/Stratis.Bitcoin.Tests/Consensus/ChainedHeaderTreeTest.cs

```diff
@@ -1284,5 +1281,45 @@ public void ChainWithMaxReorgPlusExtraHeadersIsCalled_AnotherChainIsPresented_Co
                 }
             }
         }
+
+        /// <summary>
+        /// Issue 29 @ Peer presents at least two headers. Those headers will be connected the tree. 
+        /// Then we save the first such connected block to variable X and simulate block downloaded for BOTH block.
+        /// Disconnect of the peer is then flollowed, which removes its chain from the tree. 
```

Comment:
`Then the peer is disconnected` instead of the first part of the sentence with typo